app/survey/types.tsx

```diff
@@ -26,4 +26,112 @@ export type SurveyResults = {
+};
+
+export type AgeResults = [
+    { name: "0-18"; value: number; },
+    { name: "19-24"; value: number; },
+    { name: "25-34"; value: number; },
+    { name: "35-44"; value: number; },
+    { name: "45-54"; value: number; },
+    { name: "55-64"; value: number; },
+    { name: "65+"; value: number; }
+]
+
+export type AffordFairholdResults = [
+    { name: "Yes"; value: number;},
+    { name: "Yes, but Fairhold Land Rent only, because the deposit is lower"; value: number;},
+    { name: "No, it's still too expensive"; value: number;},
+    { name: "I don't know"; value: number;}
+]
+
+export type CountryResults = {
+    [key: string]: number;
+}
+
+export type HousingOutcomesResults = [
+    { name: "Security from being evicted"; value: number;},
+    { name: "Lower cost"; value: number;},
+    { name: "Proximity to work"; value: number;},
+    { name: "Proximity to friends and family"; value: number;},
+    { name: "Proximity to good schools"; value: number;},
+    { name: "To get my money back when I sell"; value: number;},
+    { name: "To retire with low outgoings"; value: number;},
+    { name: "Lower energy bills"; value: number;},
+    { name: "Freedom to improve or repair my home myself"; value: number;},
+    { name: "Home kept in good state of repair by others"; value: number;},
+    { name: "Quality of space"; value: number;},
+    { name: "More space"; value: number;},
+    { name: "Sense of community / shared spaces within the neighbourhood"; value: number;},
+    { name: "Better public transport connections"; value: number;},
+    { name: "Walkable neighbourhood"; value: number;},
+    { name: "Lower crime / antisocial behaviour"; value: number;},
+    { name: "Being able to easily move home whenever I want to"; value: number;},
+    { name: "My home to increase in value as an investment"; value: number;},
+    { name: "Rent as a source of income"; value: number;},
+    { name: "Freedom from stress or exploitation (for example, debt, maintenance responsibilities, toxic relationships, bad landlords, or exploitative management companies)"; value: number;},
+    { name: "None of these. My current situation is fine."; value: number;},
+    { name: "Other"; value: number;}
+]
+
+export type SupportDevelopmentResults = [
+    { name: "Strongly supportive of any development"; value: number;},
+    { name: "Quite supportive of most development"; value: number;},
+    { name: "It depends"; value: number;},
+    { name: "Quite opposed to most development"; value: number;},
+    { name: "Strongly opposed to any development"; value: number;},
+    { name: "Don't know"; value: number;}
+]
+
+export type SupportDevelopmentFactorsResults = [
+    { name: "Homes that are affordable to keyworkers"; value: number;},
+    { name: "Affordable tenure homes for low-income families"; value: number;},
+    { name: "Priority given to local residents and their families (to allow downsizing, for example)"; value: number;},
```

Comment:
General question - are these mapping of the choices written in Tally?

If so, do we need them here? If we just count all responses (regardless of actual text of the answer) that should allow us to just maintain these labels once in Tally.